docs/content/intro/protocol.md

```diff
@@ -7,24 +7,26 @@ Athens builds on top of Go CLI which specifies a set of endpoints with which it
 
 The original vgo research paper on Download protocol can be found here: https://research.swtch.com/vgo-module
 
-Each of these endpoints sits on top of a module. Let's assume module `htp` authored by `acidburn`.
+Each of these endpoints sits on top of a module. Let's assume module `assert` authored by `arschles`.
 
-So for each of endpoints mentioned bellow we will assume address `acidburn/htp/@v/{endpoint}` (e.g `acidburn/htp/@v/list`)
+So for each of endpoints mentioned bellow we will assume address `arschles/assert/@v/{endpoint}` (e.g `arschles/assert/@v/list`)
```

Comment:
The changes at line 10, 12, and 18 are to be consistent with the texts and the examples. I am okay to change to a different endpoint as long as the texts and examples are consistent.

Reply:
I think we should change this back to a fake repo, only because the versions listed do not exist for `arschels/assert` and it would be nice to show multiple version for our example.

Reply:
Makes sense. I changed back to fake endpoints `acidburn/htp/@v/{endpoint}` in my latest commit.